src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -577,6 +595,18 @@ function handleSearchInputValue(): void {
   updateMatchingLineNumbers()
 }
 
+function setDefaultMatchingLineNumbers(): void {
+  isProcessingInternally.value = true
+  regExpError.value = null
+
+  matchingLineNumbers.value = Array.from(new Set(props.highlightedLineNumbers))
+  numberOfMatches.value = matchingLineNumbers.value.length
+
+  emitMatchingLinesChangeEvent()
+
+  isProcessingInternally.value = false
+}
+
```

Comment:
question: If you want to pass in a range (e.g. highlight lines 2-8) could this be accounted for instead of having to pass in each line number individually?

Reply:
We will stick to an array of line numbers for now (since that is what is expected by the highlighter) and revisit if a need for a more elegant solution arises.